### PR TITLE
Don't check interface is up if it's loopback.

### PR DESCRIPTION
**Pull request by cprivite**
Signed-off-by: Chris Privitere <cprivite@users.noreply.github.com>

### CODE REVIEW

**Comment by cprivite** [comment]
Fixes #342

**Review by thebsdbox — APPROVED** [review]
`YAY` also `/LGTM`

File: pkg/kubevip/config_manager.go [hunk 1]
```diff
@@ -205,6 +206,10 @@ func isValidInterface(iface string) error {
 		return fmt.Errorf("get %s failed, error: %w", iface, err)
 	}
 
+	if l.Attrs().Flags&net.FlagLoopback == net.FlagLoopback {
```
Comment (displague) [inline pkg/kubevip/config_manager.go:209]: I think a linter would add some space here.

Comment (cprivite) [reply pkg/kubevip/config_manager.go:209]: golangci-lint, go vet, and go fmt aren't suggesting anything. What are you thinking would be better?

Comment (cprivite) [reply pkg/kubevip/config_manager.go:209]: golangci-lint, go vet, and go fmt aren't suggesting anything. What are you thinking would be better?

Comment (thebsdbox) [reply pkg/kubevip/config_manager.go:209]: https://github.com/kube-vip/kube-vip/runs/4764985605?check_suite_focus=true#step:6:116

apparently not :D

Comment (displague) [reply pkg/kubevip/config_manager.go:209]: I stand corrected - I tried `gofumpt -l -w .` and `golangci-lint run ./... --fix` (with the enabled settings for this project, also with the tinkerbell/tink settings). No change.

**Pull request merged**